Makefile

```diff
@@ -21,6 +21,10 @@ $$($1_test_srcs): $1__%.py :
 	| sed -e "s/^/$$$$$$$$ /"
 
 .PHONY: $1_test $$($1_test_srcs)
+
+# Serialize the tests that use a local Elasticsearch instance
+$1__tests/test_search.py: $1__tests/test_indexer.py
+$1__tests/test_indexer.py: $1__tests/test_subscriptions.py
```

Comment:
Not that this needs to be fixed here, but I'm decidedly not a fan of all the magic alphabet soup that has crept into the Makefile test rules. To serialize the tests, just make a target that calls them serially:

```makefile
serial_tests:
	$(MAKE) tests/test_search.py
	$(MAKE) tests/test_indexer.py
```
 and exclude them in the wildcard parallel test target. If I were touching the Makefile today, I would rip out all of this target definition, eval, and octuple dollar sign madness.

Reply:
This change shouldn't be here. I'll remove it. Well spotted.

However, since I've pushed to this PR many times by now, I can already tell that this change does make the build more stable. So we might consider making those changes anyways, in another PR. I think I have a solution to address @ttung's [concern](https://github.com/HumanCellAtlas/data-store/pull/925#discussion_r164573215) but it would require leaving the [magic](https://github.com/HumanCellAtlas/data-store/pull/925#discussion_r165117509) in.